src/main/resources/translations/Messages_nl.properties

```diff
@@ -0,0 +1,161 @@
+# set list_header to "== &d[&6F&d]&f ==", list_row to "&6> %s" and list_separator to " &3| &b", our list will look
+# something like this:
+#
+# == &d[&6F&d]&f ==
+# &6> <player> &3| &b<player> &3| &b<player>
+# &6> <player> &3| &b<player> &3| &b<player>
+# &6> <player> &3| &b<player> &3| &b<player>
+#
+# For the most parts, translators won't need to touch this, aside from translating "Friends"
+
+friends_command_list_header = == &d[&6Vrienden&d]&f ==
+friends_command_list_row = &b%s
+friends_command_list_separator = &3, &b
+
+friends_command_no_friends = &cGeen vrienden gevonden:(
+
+# Setowner command
+
+setowner_command_changing_ownership = &9Bezig met het veranderen van de eigenaar van de blokken &c%s&9. Dit kan een tijdje duren.
+setowner_command_make_worldedit_selection = &cMaak een geldige cuboid selectie alstublieft. 
+setowner_command_usage = &9Gebruik: &b/&3%s &2<speler>
+setowner_command_usage_exclamation_point = &9Plaats een uitroepteken &b"!"&9 inplaats van een gebruikersnaam / geselecteerde blokken om het te verstoten.
+
+# Transferblocks command
+
+transferblocks_command_transferring_blocks = &9Overzetten van Uw blokken naar &c%s&9. Dit kan een tijdje duren.
+transferblocks_command_usage = &9Gebruik: &b/&3%s &2<speler>
+transferblocks_thread_complete = &9Eigenaarschap van &c%s blokken&9 is veranderd. 
+
+# Transferplayer command
+
+transferplayer_command_usage = &9Gebruik: &b/&3%s &2<speler> <doel>
+transferplayer_command_transferring_blocks = &9Overzetten blokken van &c%s&9 naar &c%s&9. Dit kan een tijdje duren.
+
+# Unfriend command
+
+unfriend_command_console_usage = &9Gebruik: &b/&3%s &2<speler> <vriend>
+unfriend_command_console_not_friends = &b%s&c is niet vrienden met &b%s.
+unfriend_command_console_no_longer_friends = &b%s&a is niet meer vrienden met &b%s.
+
+unfriend_command_usage = &9Gebruik: &b/&3%s &2<speler>
+unfriend_command_not_friends = &cU bent geen vrienden met &b%s.
+unfriend_command_no_longer_friends = &aU bent niet meer vrienden met &b%s.
+
+### IMPORTERS ###
+
+import_logging_prefix = IMPORTEREN | %s
+
+# WatchBlock importer
+
+watchblock_import_chunks_done = Segmenten geladen: %s/%s
+watchblock_import_converting_friendships = Vriendschappen overzetten…
+watchblock_import_complete = Importeren is compleet!
+watchblock_import_created_friendship = Vrienschap gemaakt: %s -> %s
+watchblock_import_disabling_watchblock = WatchBlock wordt nu uitgeschakeld. Vergeet het niet om te verwijderen anders kunnen er volgende keer problemen voorkomen.
+watchblock_import_fetched_uuid = UUID ophalen voor speler: %s
+watchblock_import_found_worlds = Gevonden werelden (%s): %s
+watchblock_import_friendships_converted = Vriendschappen overgezet.
+watchblock_import_importing_chunks = Overzetten segmenenten %s. Dit kan een tijdje duren.
+watchblock_import_loading_blocks_for_world = Blokken laden voor wereld: %s
+watchblock_import_looking_for_worlds = Zoeken voor werelden om te importeren…
```

Comment:
That's an ellipsis (`…`), not actually three dots. Make it three dots (`...`), please.